src/components/DeeplinkWrapper/index.website.js

```diff
@@ -18,12 +18,18 @@ function isMacOSWeb() {
 }
 
 function DeeplinkWrapper({children}) {
-    useEffect(() => {
-        if (!isMacOSWeb() || CONFIG.ENVIRONMENT === CONST.ENVIRONMENT.DEV) {
-            return;
-        }
+    const isUnsupportedDeeplinkRoute = useMemo(
+        () =>
+            // According to the design, we don't support unlink in Desktop app https://github.com/Expensify/App/issues/19681#issuecomment-1610353099
+            _.some([CONST.REGEX.ROUTES.UNLINK_LOGIN], (unsupportRouteRegex) => {
+                const routeRegex = new RegExp(unsupportRouteRegex);
+                return routeRegex.test(window.location.pathname);
+            }),
+        [],
+    );
```

Comment:
Why do we need to memorize this?

Reply:
1 value in array I don't think its expensive. Let's just have a definition and use it.